app/extensions/string_extensions.rb

```diff
@@ -381,12 +386,14 @@ def tl(sanitize = true)
 
   # TODO: Move somewhere that content_tag is defined
   def tp(sanitize = true)
-    '<div class="textile">'.html_safe + Textile.textilize(self, false, sanitize).html_safe + "</div>".html_safe
+    '<div class="textile">'.html_safe <<
+      Textile.textilize(self, false, sanitize).html_safe <<
```

Comment:
I'd rather there was a `textilize_safe` for that piece and then do all the other stuff with existing safe methods.

Reply:
Done.